lib/item.rb

```diff
@@ -0,0 +1,8 @@
+class Item
+  def initializer(name:, price:, stock:)
+    @name = name
+    @price = price
+    @stock = stock
+  end
```

Comment:
This part seems unnecessary, as attr_accessor does this for us, I believe.

Reply:
I wanted to be able to shorten the code when I am creating new Orders or Items.